### ISSUE TITLE
Is it worth adding aria expanded to details?

### ISSUE THREAD

**Issue by joelanman** [alphagov/govuk-frontend#4972]
## Description of the issue

This issue is now very old, is it worth adding `aria-expanded` to details to fix it in Frontend instead?

 - https://github.com/alphagov/reported-bugs/issues/29

Came up recently here:

 - https://github.com/alphagov/govuk-design-system-backlog/issues/44#issuecomment-2098352959

**Comment by romaricpascal** [comment]
Hi @joelanman, thanks for suggesting this. We currently cannot prioritise looking into this, but one thing that came to mind regarding the solution is that we'd not only need to make sure it fixes the issue for VoiceOver and Safari, but also doesn't negatively impact users with other browser/assisitive tech combinations.

**Comment by owenatgov** [comment]
We did look into this and from some rough testing, it unfortunately didn't make a difference. The root problem however _will_ be solved by https://github.com/alphagov/govuk-frontend/pull/5089
